pkg/controller/storagecluster/reconcile.go

```diff
@@ -1253,13 +1203,13 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 	const purgeOSDScript = `
 set -x
 
-osd_status=$(ceph osd tree | grep "osd.${FAILED_OSD_ID} " | awk '{print $5}') 
-if [[ "$osd_status" == "up" ]]; then 
+osd_status=$(ceph osd tree | grep "osd.${FAILED_OSD_ID} " | awk '{print $5}')
+if [[ "$osd_status" == "up" ]]; then
```

Comment:
They look like unnecessary changes

Reply:
I think these changes came from extra spaces. Fixing it.

Reply:
This is correct IMO as the previous contributor **added extra spaces**, my vscode settings remove unnecessary extra whitespace. I didn't add/removed any spaces by myself.

Reply:
Woah! Then thats good :)